numpy/doc/basics.py

```diff
@@ -260,6 +260,45 @@
 value is inside an array or not.  NumPy scalars also have many of the same
 methods arrays do.
 
+Overflow Errors
+===============
+
+The fixed size of NumPy numeric types may cause overflow errors when a value
+requires more memory than available in the data type. For example ``np.power``
+evaluates ``100 * 10 ** 8`` correctly for 64-bit integers, but gives
+1874919424 (not correct) for a 32-bit integer.
+
+    >>> np.power(100, 8, dtype=np.int64)
+    10000000000000000
+    >>> np.power(100, 8, dtype=np.int32)
+    1874919424
+
+The behaviour of NumPy and Python integer types differs significantly for
+integer overflows and may confuse users expecting NumPy integers to behave
+similar to Python's ``int``. Unlike NumPy, the size of Python's ``int`` is
+flexible. This means Python integers may expand to accomodate any integer and
+will not overflow.
+
+NumPy provides :ref: ``numpy.iinfo`` and :ref: ``numpy.finfo`` to verify the
+minimum or maximum values of NumPy integer and floating point values
+respectively ::
+
+    >>> np.iinfo(np.int) # Bounds of the default integer
+    iinfo(min=-9223372036854775808, max=9223372036854775807, dtype=int64)
```

Comment:
"Bounds of the default integer" -> "Bounds of the default integer on 64-bit platforms"

Reply:
Good pick up. Although it's also not *always* correct as a C `long` on Windows only has 32-bits:

    Python 3.6 (64-bit) interactive window [PTVS 15.9.18254.1-15.0]
    >>> import platform
    >>> platform.architecture()
    ('64bit', 'WindowsPE')
    >>> import numpy as np
    >>> np.iinfo(np.int)
    iinfo(min=-2147483648, max=2147483647, dtype=int32)

maybe -> "Bounds of default int on this system"?

Reply:
ok